app/models/order.rb

```diff
@@ -6,6 +6,10 @@ class Order <ApplicationRecord
   has_many :item_orders
   has_many :items, through: :item_orders
 
+  def self.by_status
+    order(status: :asc)
+  end
```

Comment:
I didn't know how to feel about changing the `order.status` enum default order as a solution (from one of my PRs), but looking at this query makes me feel very okay about it 👍 I like this class method!